src/Core/ConstraintValidator/Constraints/NoTags.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
This used to be in https://github.com/PrestaShop/PrestaShop/pull/20243/files PR but it's no longer needed there and needed here so I moved it.